pkg/knuu/instance.go

```diff
@@ -1198,11 +1198,9 @@ func (i *Instance) SetBandwidthLimit(limit int64) error {
 	return nil
 }
 
-// SetLatency sets the latency of the instance
-// latency in ms (e.g. 1000 for 1s)
-// jitter in ms (e.g. 1000 for 1s)
-// Currently, only one of bandwidth, jitter, latency or packet loss can be set
-// This function can only be called in the state 'Commited'
+// SetLatencyAndJitter sets both the latency and jitter of an instance in milliseconds (e.g., 1000 for 1 second).
+// With the BitTwister integration, multiple traffic shaping rules like latency and jitter can be set simultaneously.
+// This function can only be called when the instance is in the 'Started' state.
```

Comment:
as @staheri14 stated in slack, it will be very useful if we can add some examples here as well. with some explanation what user should expect.

This link provides some documentation about the latency & jitter that is used here: https://www.man7.org/linux/man-pages/man8/tc-netem.8.html

Reply:
I have made the requested changes. Please review them at your convenience.

Reply:
You're welcome! It's a pleasure contributing to this project.